cmd/usd-mailer/main.go

```diff
@@ -14,10 +14,10 @@ func main() {
 	pkg.LoadConfig()
 	database.ConnectDatabase()
 
-	database.DB.AutoMigrate(&models.Email{})
-
+	if err := database.DB.AutoMigrate(&models.Email{}); err != nil {
+		log.Fatalf("Coulnd't migrate database: %v", err.Error())
```

Comment:
Will it throw an error to not proceed with the next steps?

Reply:
Yes, I guess if migration failed the whole initialization of an app failed, not 100% sure though. Good point

Reply:
Just to make it clear, should the failed migration also considered to be a failed initialization or we can still run the app if the migration failed?

Reply:
Will your application be able to work with an unupdated database schema? If we ignore the error, then the new code will work with the old database schema, we will not know whether everything will work correctly in this case.
The correct way to deploy to the product is to migrate first, if the migration fails, then the new code will not be deployed to the server.